### PR TITLE
Update to noarch -robotframework-browser-17.4.0

### PR DESCRIPTION
**Pull request by rxm7706**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Comment by rxm7706** [comment]
@xylix @rpaframework-core 
Could you please review and merge ?
Thank you and Regards.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,45 +1,47 @@
...
+    - python >=3.8,<4.0
+    - nodejs >=18.0
     - grpcio
     - grpcio-tools
-    - robotframework
-    - robotframework-pythonlibcore
-    - typing-extensions
-    - wrapt
+    - protobuf
+    - robotframework >=6.0.1
```
Comment (osrjv) [inline recipe/meta.yaml]: Is there a reason for pinning to RF >= 6.x and not >= 5.x like in upstream?

Comment (rxm7706) [reply recipe/meta.yaml]: @osrjv This has been fixed - it should be ```>= 5.x like in upstream```.

**Pull request merged**